libfwupdplugin/fu-context.c

```diff
@@ -1001,6 +1003,60 @@ fu_context_housekeeping(FuContext *self)
 
 typedef gboolean (*FuContextHwidsSetupFunc)(FuContext *self, FuHwids *hwids, GError **error);
 
+/**
+ * fu_context_get_fde:
+ * @self: a #FuContext
+ *
+ * Gets whether FDE is in use
+ *
+ * Returns: gboolean
+ *
+ * Since: 2.05
+ **/
+gboolean
```

Comment:
Maybe make this a enum? Then we can get the *type* of fde too?

Reply:
Or add more flags rather than a new boolean?

Reply:
Yeah I like more flags instead, and that is more logical so we can get type of FDE.  Let me rework.